GeoIP2/WebServiceClient.cs

```diff
@@ -161,14 +161,16 @@ public CityResponse City(string ipAddress)
 
         private IRestClient CreateClient()
         {
-            var restClient = new RestClient("https://" + _host + "/geoip/v2.1");
-            restClient.Authenticator = new HttpBasicAuthenticator(_userId.ToString(), _licenseKey);
+            var restClient = new RestClient("https://" + _host + "/geoip/v2.1")
+            {
+                Authenticator = new HttpBasicAuthenticator(_userId.ToString(), _licenseKey)
+            };
             restClient.AddHandler("application/vnd.maxmind.com-insights+json", new JsonDeserializer());
             restClient.AddHandler("application/vnd.maxmind.com-country+json", new JsonDeserializer());
             restClient.AddHandler("application/vnd.maxmind.com-city+json", new JsonDeserializer());
             restClient.Timeout = _timeout;
 
-            restClient.UserAgent = string.Format("GeoIP2 .NET Client {0}", Version);
+            restClient.UserAgent = $"GeoIP2 .NET Client {Version}";

```

Comment:
Isn't this $ string interpolation feature new to C# 6.0? I understand it is a compiler feature and generated code will not require the latest .NET, but are we OK we making it build-able only on v6?